mathics/builtin/atomic/numbers.py

```diff
@@ -183,11 +187,17 @@ class Accuracy(Builtin):
      = Infinity
 
     >> Accuracy[F[1.3, Pi, A]]
-     = 14.8861
+     = 15.8406
```

Comment:
If this can vary, the please let's use '...' and make precise tests as pytests where we also have assertions or checks on the underlying factors that make this so.

Reply:
As you know with the MLMath stuff, I am not a fan of fragile tests because they fail unexpectedly at a time when you really don't want to start dealing with the failure.

Reply:
In this case, the "fragility" on the test allows for checking the behavior. But maybe I could surround it with `Round`.

Reply:
You could, but this is the wrong philosophy: the _primary_ purpose of these tests is to enlighten users as to what this Builtin does; testing is secondary. So complicating an example to make testing is not the right philosophy.

If you want a _test_, the add this as a pytest where you can get as elaborate as you want, like within so many digits. But better here is to _also_ add checks for the fundamental properties that the this calculation is based on. That way when that assumption breaks for whatever reason, we have pinpointed at a low level what the problem is rather than have to recall or look around in the code what Accuracy assumes that has been violated.